### PR TITLE
Add Pydantic People Page

### LINKED ISSUE #7576: Add `Pydantic People` Section to Our Docs
### Initial Checks

- [X] I have searched Google & GitHub for similar requests and couldn't find anything
- [X] I have read and followed [the docs](https://docs.pydantic.dev) and still think this feature is missing

### Description

@sydney-runkle should work on adding a `Pydantic People` or `Pydantic Contributors` section to the docs that encourages open source contributions and highlights the work of regular contributors 😄.

### Affected Components

- [ ] [Compatibility between releases](https://docs.pydantic.dev/changelog/)
- [ ] [Data validation/parsing](https://docs.pydantic.dev/usage/models/#basic-model-usage)
- [ ] [Data serialization](https://docs.pydantic.dev/usage/exporting_models/) - `.model_dump()` and `.model_dump_json()`
- [ ] [JSON Schema](https://docs.pydantic.dev/usage/schema/)
- [ ] [Dataclasses](https://docs.pydantic.dev/usage/dataclasses/)
- [ ] [Model Config](https://docs.pydantic.dev/usage/model_config/)
- [ ] [Field Types](https://docs.pydantic.dev/usage/types/) - adding or changing a particular data type
- [ ] [Function validation decorator](https://docs.pydantic.dev/usage/validation_decorator/)
- [ ] [Generic Models](https://docs.pydantic.dev/usage/models/#generic-models)
- [ ] [Other Model behaviour](https://docs.pydantic.dev/usage/models/) - `model_construct()`, pickling, private attributes, ORM mode
- [ ] [Plugins](https://docs.pydantic.dev/) and integration with other tools - mypy, FastAPI, python-devtools, Hypothesis, VS Code, PyCharm, etc.

### PR DESCRIPTION
**Pull request by sydney-runkle**
## Change Summary

Add Pydantic People page, modeled after that of FastAPI, to encourage contributions and celebrate contributors.

## Related issue number

Fix #7576

## Checklist

* [ ] The pull request title is a good summary of the changes - it will be used in the changelog
* [ ] Unit tests for the changes exist
* [ ] Tests pass on CI
* [ ] Documentation reflects the changes where applicable
* [ ] My PR is ready to review, **please add a comment including the phrase "please review" to assign reviewers**

### CODE REVIEW

File: .github/actions/people/people.py [hunk 1]
```diff
@@ -0,0 +1,612 @@
...
+from typing import Any, Container, Dict, List, Set, Union
+
+import yaml
+from github import Github
+from pydantic import BaseModel, SecretStr
+from pydantic_settings import BaseSettings
+
+github_graphql_url = "https://api.github.com/graphql"
+
+discussions_query = """
```
Comment (denisart) [inline .github/actions/people/people.py:22]: Hello

I created a package for generate a GraphQL query from python code based on pydantic.
Home page: https://github.com/denisart/graphql-query
docs: https://denisart.github.io/graphql-query/usage.html

What do you think about creating queries in graphql-query format? And will it be possible to use packages that depend on pydantic?

**Comment by denisart** [comment]
Hello

I created a package for generation a GraphQL query from python code based on pydantic.
Home page: https://github.com/denisart/graphql-query
docs: https://denisart.github.io/graphql-query/usage.html

What do you think about creating queries in graphql-query format? And will it be possible to use packages that depend on pydantic?

File: .github/actions/people/people.py [hunk 2]
```diff
@@ -0,0 +1,613 @@
...
+query Q($after: String) {
+  repository(name: "pydantic", owner: "samuelcolvin") {
```
Comment (hramezani) [inline .github/actions/people/people.py:24]: Shouldn't the owner be `pydantic` as well?
```suggestion
  repository(name: "pydantic", owner: "pydantic") {
```

File: .github/actions/people/people.py [hunk 3]
```diff
@@ -0,0 +1,613 @@
...
+    g = Github(settings.input_token.get_secret_value())
+    repo = g.get_repo(settings.github_repository)
+    question_commentors, question_last_month_commentors, question_authors = get_experts(
+        settings=settings
+    )
+    contributors, pr_commentors, reviewers, pr_authors = get_contributors(
+        settings=settings
+    )
+    authors = {**question_authors, **pr_authors}
+    maintainers_logins = {'samuelcolvin', 'adriangb', 'dmontagu', 'hramezani', 'lig', 'Kludex', 'davidhewitt', 'sydney-runkle'}
```
Comment (hramezani) [inline .github/actions/people/people.py:528]: Should we still keep `lig` as the maintainer?

File: .github/actions/people/people.py (hunk 2 above)
Comment (sydney-runkle) [reply .github/actions/people/people.py:24]: I thought so too but if I recall, the query doesn't work when I do that 🤷‍♀️.

File: .github/actions/people/people.py (hunk 3 above)
Comment (sydney-runkle) [reply .github/actions/people/people.py:528]: I'm guessing not, but we can leave him as an author in `pyproject.toml`...

**Review by hramezani — APPROVED** [review]

**Pull request merged**